lib/src/devices.dart

```diff
@@ -9,6 +9,7 @@ import 'devices/mt201.dart';
 import 'devices/mt101.dart';
 
 import 'mamac_device.dart';
+import 'package:dslink_mamac/src/devices/cf201.dart';
```

Comment:
Best practice is to not use `package` imports for files in the `lib/` directory for files of the same package. In this case it should just be a relative import path to `cf201.dart`